pretyping/reductionops.mli

```diff
@@ -274,13 +275,18 @@ type state = constr * constr Stack.t
 type state_reduction_function =
     env -> evar_map -> state -> state
 
+val stack_red_of_state_red :
+  state_reduction_function -> stack_reduction_function
+
 val pr_state : env -> evar_map -> state -> Pp.t
 
 val whd_nored_state : state_reduction_function
 
 val whd_betaiota_deltazeta_for_iota_state :
   TransparentState.t -> state_reduction_function
 
+val whd_state_gen : CClosure.RedFlags.reds -> state_reduction_function
```

Comment:
Don't export this function as well as the above `stack_red_of_state_red` wrapper, since they are only used both together. Instead you should only export a `whd_stack_gen` combining both. FTR anything remotely using the explicity stack representation delays a future change of representation.